solr/CHANGES.txt

```diff
@@ -27,7 +27,8 @@ Optimizations
 
 Bug Fixes
 ---------------------
-(No changes)
+
+* PR#2329: Fix getting fieldType by its name in FileBasedSpellChecker (Andrey Bozhko)
```

Comment:
I *believe* we normally create a JIRA for these user facing, so we can track which version of Solr it gets released.   The pattern has recently become a bit muddier...  non user facing changes sometimes skipping JIRA..   If you add a JIRA, then that makes it easier for me to merge this!

Reply:
i confirmed that yes, please open a jira...

Reply:
I submitted a request for a JIRA account, will open the issue once the account is set up